### DISCUSSION TITLE
Upgrade Jekyll

### DISCUSSION THREAD

**Discussion by ScottBouloutian** [mmistakes/so-simple-theme#327] [Q&A] [answered]
## Summary

Upgrade Jekyll

## Motivation

Fix security vulnerability:
https://nvd.nist.gov/vuln/detail/CVE-2018-17567

## Drawbacks

Potential drawback is if the upgraded version of Jekyll is not compatible with something currently being used in the theme.

**Comment by mmistakes** [comment, marked as answer]
You can update to the latest Jekyll and the theme should be compatible.

Run `gem update jekyll` and that should put you to the latest version.
